src/directives/automation/editor/panel/dpEditorPanelWhatsapp.js

```diff
@@ -265,6 +300,62 @@
         }
       }
 
+      function uploadImageSelect(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        scope.statusUploader = 'pending';
+        const imageFile = e.target.files[0];
+        if (imageFile) {
+          const maxSize = scope.multimediaType.maxSise;
+          if(imageFile.size > maxSize * 1024 * 1024) {
```

Comment:
According to this: `scope.multimediaType = multimediaConstraint[scope.selectedComponent.template.headerType || 'TEXT'];`, when the user upload a file `.txt` for example, the `if` evaluation will be `true` and the message error will be _"Oops! The size of your file is larger than..."_. In this case the message error should be: _"Oops! The extension of your file is invalid"_.

Reply:
Similar here: https://github.com/FromDoppler/doppler-automation-editor/pull/91#discussion_r1779074355.
When `scope.multimediaType` is `TEXT`, the user should not be able to upload a file.